cmd/nerdctl/login_linux_test.go

```diff
@@ -75,3 +76,102 @@ func TestLoginWithSpecificRegHosts(t *testing.T) {
 	}
 
 }
+
+func TestLoginWithPlainHttp(t *testing.T) {
+	testutil.DockerIncompatible(t)
+	base := testutil.NewBase(t)
+	reg := testregistry.NewAuthWithHttp(base, "admin", "validTestPassword", 5000)
+	testCasesForPort5000 := []struct {
```

Comment:
IMHO, We should merge `testCasesForPort80` with it.

Reply:
No, if we merge `testCasesForPort80` with `testCasesForPort5000`, we need to start/shut down all images for the test registry for each test case. I'm not sure it's right or not

Reply:
> we need to start/shut down all images for the test registry for each test case

Not necessary, just start them both at the same time.